### ISSUE TITLE
Table cell organization bug

### ISSUE THREAD

**Issue by ubidval** [Aymkdn/html-to-pdfmake#196]
I have an issue with the table cell organization when we have 2 columns that have with child columns. The children of the second column are not rendered correctly during the PDF generation with pdfmake.

The HTML code used :
```html
<table style="width: 100%;" data-pdfmake='{ "dontBreakRows": true }'>
            <th colspan="3" rowspan="2">Numéro d'ordre</th>
            <th colspan="13" rowspan="1">Titulaire</th>
            <th colspan="4" rowspan="2">Nombre de titres</th>
            <th colspan="13" rowspan="1">Bénéficiaire</th>
        </tr>

        <tr style="font-weight: bold; text-align: center;">
            <th colspan="10" rowspan="1">Nom et prénom / Dénomination sociale</th>
            <th colspan="3" rowspan="1">N° de compte</th>

            <th colspan="10" rowspan="1">Nom et prénom / Dénomination sociale</th>
            <th colspan="3" rowspan="1">N° de compte</th>
        </tr>
</table>
```
The generated PDF :
![image](https://github.com/Aymkdn/html-to-pdfmake/assets/144004246/89f69305-eeef-4fd8-9c36-73b7e4aca410)

**Comment by Aymkdn** [comment]
I need to investigate. I'll get back to you.

**Comment by Aymkdn** [comment]
I've just released v2.5.0 with a new way to deal with `colspan` and `rowspan`.

Also, make sure to use `headerRows` in `data-pdfmake` to have a better rendering:
```html
<table style="width: 100%;" data-pdfmake='{ "dontBreakRows":true, "headerRows":2 }'>
```

![image](https://github.com/Aymkdn/html-to-pdfmake/assets/946315/47498cb8-82bf-4b93-956c-52c423ca9acf)

**Comment by ubidval** [comment]
Thanks !
